### PR TITLE
New rustdoc member: jsha

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
cc @rust-lang/rustdoc 

And more precisely: cc @jsha

### CODE REVIEW

File: posts/inside-rust/2021-04-20-jsha-rustdoc-member.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+author: Guillaume Gomez
+description: "new rustdoc team member"
+team: the rustdoc team <https://www.rust-lang.org/governance/teams/dev-tools#rustdoc>
+---
+
+Hello everyone, please welcome [Jacob Hoffman-Andrews][@jsha] to the rustdoc team!
+
+Jacob Hoffman-Andrews([@jsha]) has been contributing a lot on rustdoc front-end. Thanks to him, the load of the rustdoc pages is much faster. Here is a non-exhaustive list:
+
+ * The huge search index is only loaded [when you need it]https://github.com/rust-lang/rust/pull/82310).
```
Comment (bjorn3) [inline posts/inside-rust/2021-04-20-jsha-rustdoc-member.md:13]:
```suggestion
 * The huge search index is only loaded [when you need it](https://github.com/rust-lang/rust/pull/82310).
```

Comment (GuillaumeGomez) [reply posts/inside-rust/2021-04-20-jsha-rustdoc-member.md]: Fixed, thanks!

File: posts/inside-rust/2021-04-20-jsha-rustdoc-member.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+layout: post
+title: "Jacob Hoffman-Andrews joins the Rustdoc team"
+author: Guillaume Gomez
+description: "new rustdoc team member"
+team: the rustdoc team <https://www.rust-lang.org/governance/teams/dev-tools#rustdoc>
+---
+
+Hello everyone, please welcome [Jacob Hoffman-Andrews][@jsha] to the rustdoc team!
+
+Jacob Hoffman-Andrews([@jsha]) has been contributing a lot on rustdoc front-end. Thanks to him, the load of the rustdoc pages is much faster. Here is a non-exhaustive list:
```
Comment (Manishearth) [inline posts/inside-rust/2021-04-20-jsha-rustdoc-member.md:11]:
```suggestion
Jacob Hoffman-Andrews([@jsha]) has been contributing a lot on rustdoc front-end. Thanks to him, the pageload of the rustdoc pages is much faster. Here is a non-exhaustive list of the stuff he's done recently:
```

File: posts/inside-rust/2021-04-20-jsha-rustdoc-member.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+ * The huge search index is only loaded [when you need it](https://github.com/rust-lang/rust/pull/82310).
```
Comment (Manishearth) [inline posts/inside-rust/2021-04-20-jsha-rustdoc-member.md:13]:
```suggestion
 * The huge search index is now only loaded [when needed](https://github.com/rust-lang/rust/pull/82310).
```

File: posts/inside-rust/2021-04-20-jsha-rustdoc-member.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+ * The page "flow" is [now smoother](https://github.com/rust-lang/rust/pull/82315).
```
Comment (Manishearth) [inline posts/inside-rust/2021-04-20-jsha-rustdoc-member.md:14]:
```suggestion
 * The pageload has [has been optimized a bunch](https://github.com/rust-lang/rust/pull/82315).
```

File: posts/inside-rust/2021-04-20-jsha-rustdoc-member.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+ * They also worked on the HTMLification of the toggles (show/hide documentation), reducing the amount of javascript required by rustdoc and therefore reducing even further the page load.
```
Comment (Manishearth) [inline posts/inside-rust/2021-04-20-jsha-rustdoc-member.md:15]:
```suggestion
 * Rustdoc's show/hide toggles [are being converted to pure HTML](https://github.com/rust-lang/rust/issues/83332), reducing complexity and improving pageload performance.
 ```

**Review by Manishearth — APPROVED** [review]
Some wording fixes

**Comment by GuillaumeGomez** [comment]
Thanks for the wording improvements @Manishearth ! Updated.

**Review by Manishearth — APPROVED** [review]

File: posts/inside-rust/2021-04-20-jsha-rustdoc-member.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+layout: post
+title: "Jacob Hoffman-Andrews joins the Rustdoc team"
+author: Guillaume Gomez
+description: "new rustdoc team member"
+team: the rustdoc team <https://www.rust-lang.org/governance/teams/dev-tools#rustdoc>
+---
+
+Hello everyone, please welcome [Jacob Hoffman-Andrews][@jsha] to the rustdoc team!
+
+Jacob Hoffman-Andrews([@jsha]) has been contributing a lot on rustdoc front-end. Thanks to him, the pageload of the rustdoc pages is much faster. Here is a non-exhaustive list of the stuff he's done recently:
```
Comment (Manishearth) [inline posts/inside-rust/2021-04-20-jsha-rustdoc-member.md:11]:
```suggestion
Jacob Hoffman-Andrews ([@jsha]) has been contributing a lot on rustdoc front-end. Thanks to him, the pageload of the rustdoc pages is much faster. Here is a non-exhaustive list of the stuff he's done recently:
```

**Comment by GuillaumeGomez** [comment]
I guess github is broken for the moment... Still can't see my last update. Well, we'll see at that moment. :)

**Pull request closed without merging**